lib/logstash/filters/mutate.rb

```diff
@@ -266,9 +266,13 @@ def convert(event)
       if original.nil?
         next
       elsif original.is_a?(Hash)
-        @logger.debug("I don't know how to type convert a hash, skipping",
-                      :field => field, :value => original)
-        next
+        #@logger.debug("I don't know how to type convert a hash, skipping",
```

Comment:
why is this commented out? if you add hash conversion feature, this should really go away.